docs/en/developer.md

```diff
@@ -197,23 +197,24 @@ After: framework/routes#coreroutes
 Director:
   rules:
     'tag/$ID!': 'TaxonomyDirectoryController'
-    
+
 ```
 
-In this example, any request made to `/code/<TAG>` would render a page which contains a list of pages that uses the
-Taxonomy Term specified by `<TAG>`. You can override the provided template `TaxonomyDirectory.ss` in your own theme.
+In this example, any request made to `/node/<TAG>` would render a page which contains a list of pages that uses the
+Taxonomy Term specified by `<TAG>`. You can set the path 'node/' to any thing you want e.g. 'tag', 'taxonomy' etc.
+
+You can override the provided template `TaxonomyDirectory.ss` in your own theme.
 
 Currently the following variables are available to the template;
 1. `Title` - the Taxonomy Term you are searching for
-1. `Term` - the Taxonomy Term you are searching for
```

Comment:
Removed duplicate variable "Title" can be reused.

Reply:
This is not the `Term` you are looking for...
-----

![obi-wan](https://user-images.githubusercontent.com/778003/34854058-3709998a-f79c-11e7-9b8a-5b268fffad45.jpg)